tests/JsonSchema/Command/JsonSchemaGenerateCommandTest.php

```diff
@@ -124,6 +124,22 @@ public function testArraySchemaWithReference(): void
         ]);
     }
 
+    public function testArraySchemaWithMultipleUnionTypes(): void
+    {
+        $this->tester->run(['command' => 'api:json-schema:generate', 'resource' => 'ApiPlatform\Tests\Fixtures\TestBundle\Entity\Issue6212\Nest', '--type' => 'output']);
+        $result = $this->tester->getDisplay();
+        $json = json_decode($result, associative: true);
+
+        $this->assertEquals($json['definitions']['Nest.jsonld']['properties']['owner']['anyOf'], [
+            ['$ref' => '#/definitions/Wren.jsonld'],
+            ['$ref' => '#/definitions/Robin.jsonld'],
+            ['type' => 'null'],
+        ]);
+
+        $this->assertArrayHasKey('Wren.jsonld', $json['definitions']);
+        $this->assertArrayHasKey('Robin.jsonld', $json['definitions']);
```

Comment:
I think this can be simply moved to `testArraySchemaWithReference`, as `testArraySchemaWithMultipleUnionTypes` is doing approximately the same

Reply:
I'm very willing to make changes, but am a bit confused as `Issue5793` resources don't have multiple type unions … or am I missing the obvious here?